src/main/kotlin/no/nav/familie/ef/sak/vedtak/uttrekk/UttrekkArbeidssøkerDto.kt

```diff
@@ -0,0 +1,27 @@
+package no.nav.familie.ef.sak.vedtak.uttrekk
+
+import java.time.LocalDateTime
+import java.time.YearMonth
+import java.util.UUID
+
+data class UttrekkArbeidssøkereDto(
+        val årMåned: YearMonth,
+        val antallTotalt: Int,
+        val antallKontrollert: Int,
+        val arbeidssøkere: List<UttrekkArbeidsssøkerDto>
+)
+
+data class UttrekkArbeidsssøkerDto(
+        val id: UUID,
+        val fagsakId: UUID,
+        val behandlingIdForVedtak: UUID,
+        val kontrollert: Boolean,
+        val tidKontrollert: LocalDateTime
+)
+
+fun UttrekkArbeidssøkere.tilDto() =
+        UttrekkArbeidsssøkerDto(id = this.id,
+                                fagsakId = this.fagsakId,
+                                behandlingIdForVedtak = this.vedtakId,
+                                kontrollert = this.kontrollert,
+                                tidKontrollert = this.sporbar.endret.endretTid)
```

Comment:
Å bruke endretTid her blir nog feil. Endret tid blir alltid satt når man oppretter entiteten i databasen, sån att vi har alltid en verdi der.
Forslag på løsninger? Ha ett eget felt for tidKontrollert?